### DISCUSSION TITLE
create autoscope

### DISCUSSION THREAD

**Discussion by TylerWhat** [IceHacks/SurvivCheatInjector#768] [Ideas]
**Describe the feature you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by TylerWhat** [comment]
System will help you turn the scope, depend where the enemy was

**Comment by humphreygaming** [comment]
I'm assuming this is something like variable scope changing?

**Comment by iBLiSSIN** [comment]
Not important

**Comment by IceHacks** [comment]
Zooming in just enough to see enemy? I can see the use for this. If you have a 15x scope equiped, fighting someone right next to you can be challenging.

**Comment by AsiaGuy** [comment]
auto scope mean it will auto zoom enough to see the enemy, right?
and it will zoom back if the enemy is too close :/

**Comment by TylerWhat** [comment]
> 自动瞄准镜意味着它会自动缩放到足以看到敌人的程度，对吗？ 如果敌人太近，它会放大：/

yes!

**Comment by TylerWhat** [comment]
> Zooming in just enough to see enemy? I can see the use for this. If you have a 15x scope equiped, fighting someone right next to you can be challenging.

yes

**Comment by TylerWhat** [comment]
> I'm assuming this is something like variable scope changing?

yes, kind of

**Comment by ghost** [comment]
Yeah this should be implemented into the cheat 💯 .

**Comment by IceHacks** [comment]
Okay, I will start working on this
